Firebase/Auth/Source/FIRUser.m

```diff
@@ -829,13 +829,26 @@ - (void)getIDTokenResultForcingRefresh:(BOOL)forceRefresh
   });
 }
 
+/** @fn parseIDToken:error:
+    @brief Parses the provided IDToken and returns an instance of FIRAuthTokenResult containing
+        claims obtained from the IDToken.
+
+    @param token The raw text of the Firebase IDToken encoded in base64.
+    @param error An out parameter which would contain any error that occurs during parsing.
+    @return An instance of FIRAuthTokenResult containing claims obtained from the IDToken.
+
+    @remarks IDToken returned from the backend in some cases is of a length that is not a multiple
+        of 4. In these cases this function pads the token with as many "=" characters as needed and
+        then attempts to parse the token. If the token cannot be parsed an error is returned via the
+        "error" out parameter.
+ */
```

Comment:
If this is public API, this comment should also be added to the public header.

Reply:
It isn't a public API, we just want to be clear about it's functionality since it is a bit tricky.